features/step_definitions/buyer_steps.rb

```diff
@@ -138,6 +138,22 @@ def login_form
   step 'the current domain is foo.3scale.localhost'
 end
 
+When "the developer tries to log in" do
+  username = @buyer.admins.first.username
+  set_current_domain("foo.3scale.localhost")
+  visit path_to('the login page')
+  fill_in('Username or Email', with:  username)
+  fill_in('Password', with: "supersecret")
+  click_button('Sign in')
+end
+
+Then "the developer login attempt failed" do
```

Comment:
1. I'd say "login attempt fails", "failed" reads weird in a scenario
2. Shouldn't we be more specific, and mention recaptcha in the step, and also verify the recaptcha error in flash message?

Reply:
I renamed the step, but I couldn't make it more specific. I was trying to find the recaptcha error in the resulting screen, but the error doesn't appear because the recaptcha method which adds the error is stubbed. I detect the login failure by checking we are in the login screen again but having `/session` as URL. AFAIK, the only way to see the login screen under `/session` is after a failed login attempt. I hope that's good enough.